spec/models/integration_customers/hubspot_customer_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe IntegrationCustomers::HubspotCustomer, type: :model do
+  subject(:hubspot_customer) { build(:hubspot_customer) }
+
+  describe '#targeted_object' do
+    let(:targeted_object) { Integrations::HubspotIntegration::TARGETED_OBJECTS.sample }
+
+    it 'assigns and retrieve a setting' do
```

Comment:
No blocker here, just an improvement suggestion, I think we could use a shared example to test all these assessors

Reply:
In this particular case only Hubspot customers have `targeted_object` and `email` settings.

Reply:
@ivannovosad my point was about settings_accessors in general. ;)